exotica_python/src/Exotica.cpp

```diff
@@ -598,9 +598,38 @@ PYBIND11_MODULE(_pyexotica, module)
     kdlFrame.def("getFrame", [](KDL::Frame* me) { return getFrame(*me); });
     kdlFrame.def("inverse", (KDL::Frame(KDL::Frame::*)() const) & KDL::Frame::Inverse);
     kdlFrame.def("__mul__", [](const KDL::Frame& A, const KDL::Frame& B) { return A * B; }, py::is_operator());
+    kdlFrame.def_readwrite("p", &KDL::Frame::p);
     py::implicitly_convertible<Eigen::MatrixXd, KDL::Frame>();
     py::implicitly_convertible<Eigen::VectorXd, KDL::Frame>();
 
+    py::class_<KDL::Vector>(module, "KDLVector")
+        .def(py::init())
+        .def(py::init<double, double, double>(),
+             py::arg("x") = 0,
+             py::arg("y") = 0,
+             py::arg("z") = 0)
+        .def("x", [](KDL::Vector& v) -> double& { return v[0]; })
+        .def("y", [](KDL::Vector& v) -> double& { return v[1]; })
+        .def("z", [](KDL::Vector& v) -> double& { return v[2]; })
+        .def_static("Zero", &KDL::Vector::Zero);
+
+    py::class_<KDL::RotationalInertia>(module, "KDLRotationalInertia")
+        .def(py::init<double, double, double, double, double, double>(),
+             py::arg("Ixx") = 0,
+             py::arg("Iyy") = 0,
+             py::arg("Izz") = 0,
+             py::arg("Ixy") = 0,
+             py::arg("Ixz") = 0,
+             py::arg("Iyz") = 0)
+        .def_static("Zero", &KDL::RotationalInertia::Zero);
+
+    py::class_<KDL::RigidBodyInertia>(module, "KDLRigidBodyInertia")
+        .def(py::init<double, KDL::Vector&, KDL::RotationalInertia&>(),
+             py::arg("m") = 0,
+             py::arg("oc") = KDL::Vector::Zero(),
```

Comment:
Is this the center of gravity? So ``CoG`` or ``CoM``?

Reply:
`oc` is a vector from refernce point to `CoG`.

I've taken those variable names directly from the C++ header:
> This constructor creates a cartesian space inertia matrix, the arguments are the mass, the vector from the reference point to cog and the rotational inertia in the cog.